config/crd/kustomization.yaml

```diff
@@ -5,17 +5,20 @@ resources:
 - bases/api.astarte-platform.org_astartes.yaml
 - bases/api.astarte-platform.org_astartevoyageringresses.yaml
 - bases/api.astarte-platform.org_flows.yaml
+- bases/ingress.astarte-platform.org_astartedefaultingresses.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
 - patches/webhook_in_astartes.yaml
 - patches/webhook_in_astartevoyageringresses.yaml
 - patches/webhook_in_flows.yaml
+#- patches/webhook_in_astartedefaultingresses.yaml
```

Comment:
Why is this commented? This is also probably the reason why tests fail

Reply:
This is commented as it is needed only if you have a conversion webhook in place, and it is not our case as the ADI has only one API version available at the moment.